desktop/cmp/contextmenu/ContextMenu.ts

```diff
@@ -27,26 +41,16 @@ export const [ContextMenu, contextMenu] = hoistCmp.withFactory({
     displayName: 'ContextMenu',
     memo: false, model: false, observer: false,
 
```

Comment:
we have been typically making the factory itself generic -- why the difference here.